doc/glossary.rst

```diff
@@ -171,6 +176,11 @@ general neuroimaging concepts. If you think a term is missing, please consider
         diagrams of approximate sensor positions in top-down diagrams of the head,
         so-called topographies or topomaps).
 
+    LCMV beamformer
+        Linearly constrained minimum variance beamformer, which attempts to
+        estimate activity for a given source while suppressing cross-talk from
+        other regions, see :func:`mne.beamformer.make_lcmv`.
```

Comment:
@britta-wstnr @agramfort thanks for the comments, pushed a commit to address them. In the example I linked to `:term:`s instead of giving the definitions there, which caused me to add `LCMV` and `DICS` to our `glossary.rst`. @britta-wstnr can you just check to see if this is okay? If it's not, let me know how to modify it.

Reply:
@britta-wstnr I'll merge this once CIs come back happy, but feel free to open a PR to correct the wording

Reply:
@larsoner, thanks it is on my list to look at the glossary also with the beamformer tutorial I am writing!

Reply:
FYI @britta-wstnr if you can get to the tutorial in the next couple of weeks it can make it into 0.20!

Reply:
I will do all I can! It has been on my to do list for way too long already anyway :see_no_evil:  ... the workload has been high the last couple months.